desc/compute/utils.py

```diff
@@ -685,7 +726,7 @@ def _get_grid_surface(grid, surface_label):
     grid : Grid
         Collocation grid containing the nodes to evaluate at.
     surface_label : str
-        The surface label of rho, theta, or zeta.
+        The surface label of rho, poloidal, or zeta.
```

Comment:
Rest of changes in this file just make sure stuff works when different strings are given for the same surface label.

Reply:
I'm trying to make the grid stuff agnostic to the poloidal label, since grids may be composed of rho, alpha, zeta coordinates instead of rho, theta, zeta. I don't want to write `grid.num_theta` when I really mean `grid.num_alpha`, and likewise it gets confusing to write logic like `integrate over theta` when I may want to do `integrate over alpha`.